tests/minifier.js

```diff
@@ -1962,6 +1962,22 @@
     equal(minify(input, options), output);
   });
 
+  test('ignore synthetic tags', function() {
+    equal(minify('<p id=""class=""title="">x', { ignoreSyntheticTags: true }), '<p id="" class="" title="">x');
```

Comment:
I made that term up, and I'm notoriously bad at naming things :sweat_smile: 

`autoGenerated` sounds generic enough for possible further expansion - though in this case the default would be `true`, i.e. it'd be the opposite sense to the current setting. Is that okay?